contracts/src/structs/CrossNet.sol

```diff
@@ -104,3 +104,12 @@ struct ReceiptMsg {
     // TODO: In the future we may include here events and other
     // feedback information.
 }
+
+struct ApplyMsgRet {
+    /// The ret bytes from message execution
```

Comment:
What is ret?

I think naming can be improved here. For me, even the comment "The ret bytes from message execution" says not much. Probably, ret is returned?

Reply:
sounds good, will do so. I used `ret` because it's in the source code. I will make it more expressive.